packages/documentation/src/pages/forms/available-features-and-usage-guidelines.mdx

```diff
@@ -1,3 +1,8 @@
+---
+title: Available form features and usage guidelines
+tags: footer, progress bar, title, date, alerts, radio button group, checkbox group, required field, contexual error message, dupcliate field validation, sequential duplicate form groups, review page, required checkbox before submission, remote error, event monitoring
```

Comment:
since we're only showing matching tags, it's okay to have a lot of tags in a document 
![Screen Shot 2019-05-02 at 12 00 58](https://user-images.githubusercontent.com/4998130/57096223-fcfe4e00-6cd1-11e9-9859-061b10f0ff47.png)